client/src/config/keycloakConfig.ts

```diff
@@ -0,0 +1,15 @@
+const token = localStorage.getItem('token');
+const refreshToken =localStorage.getItem('refreshToken');
+
+export const keycloakConfig : { initConfig: Keycloak.KeycloakInitOptions, auth?: Keycloak.KeycloakConfig } = {
+  initConfig: {
+    onLoad: 'check-sso',
```

Comment:
[Optional] Consider login-required instead of sso

```suggestion
    onLoad: 'login-required',
```

I see that now they enable silent check sso that doesn't do redirect. 
Genereraly not so sure how redirects work in PWA conctext.
More info: https://github.com/keycloak/keycloak-documentation/blob/master/securing_apps/topics/oidc/javascript-adapter.adoc